prometheus-exporters-formula/prometheus-exporters/files/node-exporter-config.Debian

```diff
@@ -102,8 +102,7 @@ ARGS="{{ salt['pillar.get']('node_exporter:args', '') }}"
 #  --collector.stat          Enable the stat collector (default: enabled).
 #  --collector.supervisord   Enable the supervisord collector (default:
 #                            disabled).
-#  --collector.systemd       Enable the systemd collector (default:
-#                            disabled).
+#  --collector.systemd       Enable the systemd collector (default: enabled).
```

Comment:
Do we want to enable the systemd collector by default? I think by default by upstream (https://github.com/prometheus/node_exporter#disabled-by-default) and we do not explicitly enable it in the SUSE version.

Reply:
No particular reason on my side: I just took the upstream Ubuntu configuration not realizing that they are deviating from the prometheus upstream here. From my perspective we could stick to the Ubuntu defaults though (for Ubuntu systems) in order to avoid the continuous maintenance of shared defaults? This should work as long as our pre-defined dashboards only rely on metrics that are enabled by all vendors.